[CUBRIDMAN-4] english manual for CBRD-23608 (TDE) (#249)
CUBRID/cubrid-manual: en/sql/schema/table_stmt.rst

    
    csql> ;sc tbl

.. _create-tde-table:

Table Encryption (TDE)
^^^^^^^^^^^^^^^^^^^^^^

You can encrypt a table as follows. For more information on TDE encryption, see :ref:`tde`.

.. code-block:: sql

    CREATE TABLE enc_tbl (a INT, b INT) ENCRYPT = AES;

> You can specify **AES** and **ARIA** as the encryption algorithm. If omitted as follows, the encryption algorithm specified by the system parameter **tde_default_algorithm** is used. The default value is **AES**.

Reviewer: ```suggestion
You can specify **AES** or **ARIA** as the encryption algorithm. If omitted as follows, the encryption algorithm specified by the system parameter **tde_default_algorithm** is used. The default value is **AES**.
```

.. code-block:: sql

    CREATE TABLE enc_tbl (a INT, b INT) ENCRYPT;

The encryption information is not inherited.

CREATE TABLE LIKE
-----------------

> You can create a table that has the same schema as an existing table by using the **CREATE TABLE ... LIKE** statement. Column attribute, table constraint, index, and encryption information are replicated from the existing table. An index name created from the existing table changes according to a new table name, but an index name defined by a user is replicated as it is. Therefore, you should be careful at a query statement that is supposed to use a specific index created by using the index hint syntax(see :ref:`index-hint-syntax`).

Reviewer 2: ```suggestion
You can create a table with the same schema as an existing table by using the **CREATE TABLE ... LIKE** statement. Column attribute, table constraint, index, and encryption information are replicated from the existing table. An index name created from the existing table changes according to a new table name, but an index name defined by a user is replicated as it is. Therefore, you should be careful with a query statement that is supposed to use a specific index created by using the index hint syntax(see :ref:`index-hint-syntax`).
```
